shared/api/system.go

```diff
@@ -106,7 +106,7 @@ type SystemUpdatesPut struct {
 	// the updates from the configured source is skipped. This is mainly useful
 	// for development and since it only affects the first execution after the
 	// start of the daemon, this setting is not available thought the JSON API.
-	SourcePollSkipFirst bool `json:"-" yaml:"source_skip_first_update"`
+	SourcePollSkipFirst bool `json:"source_skip_first_update" yaml:"source_skip_first_update"`
```

Comment:
Not sure about this change. I think, this config option is mainly useful for development and therefore I would not expose this to the update API. Therefore I added the `json:"-"` intentionally.

Reply:
Maybe development options should be moved to env variables so we keep a clear mapping between those structs and what's available in the API?